core/bin/dust_api.rs

```diff
@@ -1182,8 +1182,8 @@ async fn data_sources_tokenize(
                 None,
             ),
             Some(ds) => {
-                let config = ds.config().clone();
-                let llm = provider(config.provider_id).llm(config.model_id);
+                let embedder_config = ds.default_embedder_config().clone();
+                let llm = provider(embedder_config.provider_id).llm(embedder_config.model_id);
```

Comment:
Not related to this PR but this is off.

Should be `embedder` not `llm`. Should be equivalent for OpenAI.

Reply:
Track here: https://github.com/dust-tt/dust/issues/5296.